Upgrade wds to npm version

msecret opened the pull request:
I think it should only update patch updates, we shoudl decide when to do minor updates because they are often breaking

Commit:
Upgrade WDS to latest 0.10.0, fix things

Fixes required:
- Reset base `<a>` tags to not have underlined style, will still have
underlined style in content.
- Added rule about button font size

Commit:
Revert gemfile changes

Commit:
Add rule back about content a tags

As they should be underlined. This was removed from standards.

msecret commented:
@juliaelman take a look

Commit:
Broken sidenav rule

A rule was added to `ul` for left margin, so had to reset this on the
sidenav.

Commit:
Removed WDS build steps from travis build

It's no longer required to build the WDS when we're on the npm version
because the compiled files will be there.

Commit:
Fix linting errors

Commit:
Fix font paths for dashboard.

Commit:
Fix various broken things on dashboard

Commit:
Fix code segments on docs

juliaelman commented:
Looks good and thanks for making those changes!

One more thing I found in relation to list items. You can see that there is a `margin-left` set at `0.9em` for all `ul` and `ol` items [here](https://github.com/18F/web-design-standards/blob/v0.10.0/src/stylesheets/elements/_list.scss#L3). There was recently [an update made](https://github.com/18F/web-design-standards/blob/staging/src/stylesheets/elements/_list.scss#L3) to remove this. Thinking it would be good to add a style to cancel this out for now, until another upgrade of the WDS is made to `cg-style`.

Commit:
Implement some rules to override future WDS rules

These rules are coming in the next WDS release so I wanted to override
because those rules break things.

Commit:
Override .usa-disclaimer margins

Needed to reset a top margin so that the two
aspects of the disclaimer would vertically
align.

Commit:
Upgrade node to match cg-dashboard

When doing local development, it is helpful to have nvm
reference the same version of node so as to make npm
linking easier.

jeremiak commented:
@msecret Could you review my last few commits? I upgraded node and npm to be the same version of cg-dashboard so that `npm link` is easier to think about and get right.

juliasolorzano commented on src/css/base/lists.scss:
```diff
@@ -0,0 +1,6 @@
+
+ul,
+ol {
+  margin: 1rem 0 1rem 0.9em;
```
Should we change this value to `@include margin(1em null);` to match the upcoming fix?

msecret replied:
I'm keeping it as this because `@include margin(1em null);` breaks a bunch of things I don't want to break right now.

msecret commented on src/css/override_components.scss:
```diff
@@ -8,12 +8,17 @@
   }
 }
-
+//
```
This should be removed.

Commit:
Remove unused comment
